Paybox/System/Base/Request.php

```diff
@@ -147,10 +148,10 @@ public function getForm($options = array())
         $options['csrf_protection'] = false;
 
         $parameters = $this->getParameters();
-        $builder = $this->factory->createNamedBuilder('', 'form', $parameters, $options);
+        $builder = $this->factory->createNamedBuilder('', Type\FormType::class, $parameters, $options);
```

Comment:
Two problem here:

* This is not SF 2.7 compatible.
* `::class` notation is not compatible with PHP <5.5.

Please use `method_exists` for conditional parameters.